content/command-line/concepts/bash/terms/input-output-redirection/input-output-redirection.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Bash: Input-Output Redirection'
+Description: 'Redirection allows the modification of the destination of standard input and standard output in a shell.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Redirection'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+Redirection is a form of input-output manipulation in a shell program such as [Bash](https://www.codecademy.com/resources/docs/command-line/bash). Redirection allows changing the values of `stdout` and `stdin` in a shell to a different location, i.e., a file can be read to `stdin` instead of taking input from keyboard and the output can be written to a file instead of going to terminal which is the default value of `stdout` in most cases.
```

Comment:
This can be simplified for better readability like this - 

```suggestion
In shell programs such as [Bash](https://www.codecademy.com/resources/docs/command-line/bash), **redirection** is a powerful mechanism that allows control over where input comes from and where output goes. By default, commands read input (`stdin`) from the keyboard and send output (`stdout`) to the terminal. However, redirection makes it possible to read from files, write to files, or even discard output altogether.
```